src/components/Autocomplete/components/ComboBox/tests/ComboBox.test.tsx

```diff
@@ -378,6 +378,22 @@ describe('<ComboBox/>', () => {
     });
   });
 
+  describe('disabled', () => {
+    it('does not set Popover to active if TextField is disabled', () => {
```

Comment:
I might also consider adding a similar to AutoComplete. Just for the sake of documenting how it's done. Also to check it's not focusable maybe.